Src/LinearSolvers/MLMG/AMReX_MLNodeLaplacian.cpp

```diff
@@ -1089,6 +1089,17 @@ MLNodeLaplacian::restriction (int amrlev, int cmglev, MultiFab& crse, MultiFab&
 
     const auto& stencil = m_stencil[amrlev][cmglev-1];
 
+    bool regular_coarsening = true; int idir = 2;
+    if (amrlev == 0 and cmglev > 0) {
+        regular_coarsening = mg_coarsen_ratio_vec[cmglev-1] == mg_coarsen_ratio;
+        IntVect ratio = (amrlev > 0) ? IntVect(2) : mg_coarsen_ratio_vec[cmglev-1];
```

Comment:
This is inside `if (amrlev == 0 and ..)` so it's not possible `amrlev > 0`.

Reply:
Sure, it can be left with just cmglev > 0.